www/src/pages/home/editDedupSite/EditDedupMineralSite.tsx

```diff
@@ -60,6 +73,37 @@ export const EditDedupMineralSite = observer(({ dedupSite, commodity }: EditDedu
 
   const columns = useMemo(() => {
     return [
+      {
+        title: "User",
+        key: "user",
+        render: (_: any, site: MineralSite) => {
+          var username = site.createdBy[0]?.split("/").pop() || "Unknown";
+          if (username === "umn" || username === "sri" || username === "inf") {
+            username = "Database"
+          }
+          const color = getUserColor(username);
+          var fullName = site.createdBy[0]?.split("/").pop();
+          if (fullName === "umn" || fullName === "sri" || username === "inf") {
+            fullName = "Database"
+          } const confidence = 0.85;
```

Comment:
I updated the created by to this new format: https://minmod.isi.edu/users/s/, if it is a system, /u/ if it is a user.

Use that instead of hard code the usernames.

Also, use System instead of database.

For confidence, use the new 'score' field.